pkg/cmd/server/api/v1/conversions.go

```diff
@@ -62,6 +62,16 @@ func init() {
 				obj.ExecHandlerName = DockerExecHandlerNative
 			}
 		},
+		func(obj *ServingInfo) {
+			if len(obj.BindNetwork) == 0 {
+				obj.BindNetwork = "tcp4"
```

Comment:
It's misleading but I don't want to have different places.  Certain config can ignore bind network now, and we can consider it a bug.